src/index.tsx

```diff
@@ -806,33 +824,56 @@ function generateDraw(element: ExcalidrawElement) {
         element.height / 2,
         element.width,
         element.height,
-        { stroke: element.strokeColor, fill: element.backgroundColor }
+        {
+          stroke: element.strokeColor,
```

Comment:
Not exactly sure what happened but I refreshed and loaded a scene from a previous version and all the shapes are now invisible. We need to figure out how to get minimal backwards compatibility.

After:
![image](https://user-images.githubusercontent.com/197597/71789068-39124d00-2fdd-11ea-959d-5e6a55504b1d.png)


Before:
![image](https://user-images.githubusercontent.com/197597/71789073-3dd70100-2fdd-11ea-8fa7-2718b09fd34a.png)